tool/publish.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -x #echo on
+echo "Editing .gitignore to comment out build directory"
+
+pushd packages/devtools
+perl -pi -e "s/^build\/\$/\# build\//g" .gitignore
+
+pub get
+
+pub run webdev build
```

Comment:
perhaps move this before the .gitignore file modification in case it fails?

Reply:
done